### PR TITLE
:pencil: Video Celebration post

### PR DESCRIPTION
**Pull request by katialira**
Closes # .

Post about the 2020 Video Celebration on vIDhug

### CODE REVIEW

**Review by jefftriplett — APPROVED** [review]
LGTM 🎉

File: _posts/2020-10-21-djangocon-us-2020-video.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+author: Katia Lira
+category: General
+date: 2020-10-21 11:00:00
+layout: post
+image: /static/img/blog/announcing-2019.png
```
Comment (drewbrew) [inline _posts/2020-10-21-djangocon-us-2020-video.md]: Is this the right image that we want?

(Just double-checking)

Comment (drewbrew) [reply _posts/2020-10-21-djangocon-us-2020-video.md]: image

Update: looks like the image doesn't render, so I guess it's okay?

Comment (drewbrew) [reply _posts/2020-10-21-djangocon-us-2020-video.md]: Update 2: it appears in slack previews, so we don't want that image.

image

Comment (katialira) [reply _posts/2020-10-21-djangocon-us-2020-video.md]: Fixed!

**Pull request merged**